test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -170,3 +192,62 @@ test.requestHooks(mockChallengeEmail)('should navigate to email challenge page',
   const challengeFactorPage = new ChallengeFactorPageObject(t);
   await t.expect(challengeFactorPage.getPageTitle()).eql('Verify with your email');
 });
+
+test.requestHooks(mockChallengeOVTotp)('should load select authenticator list with okta verify options', async t => {
+  const selectFactorPage = await setup(t);
+  await t.expect(selectFactorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
+  await t.expect(selectFactorPage.getFormSubtitle()).eql('Select from the following options');
+  await t.expect(selectFactorPage.getFactorsCount()).eql(4);
+
+  await t.expect(selectFactorPage.getFactorLabelByIndex(0)).eql('Use Okta FastPass');
+  await t.expect(await selectFactorPage.factorDescriptionExistsByIndex(0)).eql(false);
+  await t.expect(selectFactorPage.getFactorIconClassByIndex(0)).contains('mfa-okta-verify');
+  await t.expect(selectFactorPage.getFactorSelectButtonByIndex(0)).eql('Select');
+
+  await t.expect(selectFactorPage.getFactorLabelByIndex(1)).eql('Get a push notification');
+  await t.expect(await selectFactorPage.factorDescriptionExistsByIndex(1)).eql(false);
+  await t.expect(selectFactorPage.getFactorIconClassByIndex(1)).contains('mfa-okta-verify');
+  await t.expect(selectFactorPage.getFactorSelectButtonByIndex(1)).eql('Select');
+
+  await t.expect(selectFactorPage.getFactorLabelByIndex(2)).eql('Enter a code');
+  await t.expect(await selectFactorPage.factorDescriptionExistsByIndex(2)).eql(false);
+  await t.expect(selectFactorPage.getFactorIconClassByIndex(2)).contains('mfa-okta-verify');
+  await t.expect(selectFactorPage.getFactorSelectButtonByIndex(2)).eql('Select');
+
+  await t.expect(selectFactorPage.getFactorLabelByIndex(3)).eql('Okta Password');
+  await t.expect(await selectFactorPage.factorDescriptionExistsByIndex(3)).eql(false);
+  await t.expect(selectFactorPage.getFactorIconClassByIndex(3)).contains('mfa-okta-password');
+  await t.expect(selectFactorPage.getFactorSelectButtonByIndex(3)).eql('Select');
+
+  // signout link at enroll page
+  await t.expect(await selectFactorPage.signoutLinkExists()).ok();
+  await t.expect(selectFactorPage.getSignoutLinkText()).eql('Sign Out');
+
+});
+
+test.requestHooks(mockChallengeOVTotp)('should navigate to okta verify totp page', async t => {
```

Comment:
For the rest of test cases, I think it's more important to assert the request body than response.
My reasoning is as long as the request body is correct, we would receive expected response in real world otherwise it is problem from backend.
With mocking response, even request body is incorrect, the test still success but will fail in practice.